src/bodywork/k8s/workflow_jobs.py

```diff
@@ -185,6 +183,70 @@ def create_workflow_cronjob(cron_job: k8s.V1Job) -> None:
     )
 
 
+def update_workflow_cronjob(
+    namespace: str,
+    name: str,
+    schedule: str = None,
```

Comment:
`Optional[str] = None` + check the args below.

Might be worth doing a `ctrl+F` on the codebase, just in case I've missed any. I can't believe that MyPy hasn't complained.